cfme/tests/configure/test_log_depot_operation.py

```diff
@@ -295,10 +297,41 @@ def _clear_ftp():
         ftp.recursively_delete()
 
     # Start the collection
+    collect_time = datetime.utcnow() - timedelta(hours=4)  # time on dropbox is UTC-4
     configured_depot.collect_all()
     # Check it on FTP
-    check_ftp(ftp=log_depot.ftp, server_name=appliance.server.name,
-              server_zone_id=appliance.server.zone.id, check_ansible_logs=True)
+    if log_depot.protocol != 'dropbox':
+        check_ftp(ftp=log_depot.ftp, server_name=appliance.server.name,
+                  server_zone_id=appliance.server.zone.id, check_ansible_logs=True)
+    else:  # check for logs on dropbox
+        username = conf.credentials['rh_dropbox']['username']
+        password = conf.credentials['rh_dropbox']['password']
+
+        dropbox = FTP(host='flopbox.corp.redhat.com', user=username, passwd=password)
```

Comment:
This hostname shouldn't be hardcoded into the test, and should be pulled from yamls.

Reply:
After some discussion, it appears this is auto-filled when running CFME, but isn't a relevant test for upstream.

1. Skip/uncollect upstream when the `log_depot.protocol` is dropbox (in fixture? comment above)
2. Pull this host from the UI if possible, instead of hard coding in test.